.github/workflows/python.yml

```
@@ -0,0 +1,32 @@
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8 pytest
+
+    - name: Run flake8
+      run: |
+        flake8 .
+
+    - name: Run pytest
+      run: |
+        pytest
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)